kinder/ci/tools/update-workflows/config.yaml

```diff
@@ -186,3 +186,6 @@ jobGroups:
     - ./templates/workflows/dryrun-tasks.yaml
   jobs:
   - kubernetesVersion: latest
+  - kubernetesVersion: 0
+  - kubernetesVersion: -1
+  - kubernetesVersion: -2
```

Comment:
https://github.com/kubernetes/kubeadm/blob/2c55ced7317ae3def1167775d2dfb0253a403e67/kinder/ci/workflows/dryrun-tasks.yaml#L11

There is a var `upgradeVersion` in dryrun-tasks.yaml.
Should we set it as well?

Reply:
Yeah, we should set it. Only the `latest` Kubernetes version does not need to be configured.